torch/_torch_docs.py

```diff
@@ -595,10 +595,12 @@ def merge_dicts(*dicts):
            r"""
 atan2(input, other, out=None) -> Tensor
 
-Returns a new tensor with the arctangent of the elements of :attr:`input`
-and :attr:`other`.
+Element-wise arc tangent of :math:`\text{{input}}_{{i}} / \text{{other}}_{{i}}`
+with consideration of the quadrant. Returns a new tensor with the signed angles
+in radians between vector :math:`(\text{{other}}_{{i}}, \text{{input}}_{{i}})`
```

Comment:
Adding a `(note the reverse order of other and input here)` after the coordinate would be useful no?